### ISSUE TITLE
Add a missing "About", "Contact" "Information" pages to let users know important information about Twake

### ISSUE THREAD

**Issue by Facyla** [linagora/twake-on-matrix#1769]
It is nice to have ethical values, and to implement them, but it is also important to let the users know how the software they use (or will potentially use) is built on, at which stage of development it is, what are the upcoming features, and provides them with means to provide feedback and to contact the operators of the service. 

With these goals in mind, i'd suggest to add at least one menu entry (which could be in the "user" submenu, or into the "apps" menu - which BTW should also be streamlined into the same interface), with names likes "About", "Contact", "Licenses", "Our values" or other headers, and would provides links and details on the following topics: 

- the software license + link to project page(s) + the used OSS components (and their license) + link to the project contributors (or some stats + links) : because it demonstrates/proves that it really is a fully free and open source software. Explaining to non-jurists what this means and implies, and which serious issues it solves (or which threats it contribute to mitigate) is much more than a nice-to-have: it is also a reinsurance, and the most valid illustration proof that justifies the link that is not-so-obvious for everyone between OSS, democracy, digital autonomy, etc.
- a page about the values that the main contributor/initiator of that project struggles to defend, and why these choices have been made
- a contact and feedback page : should be targeted at end user, but also easily customisable, as there are at least 2 main use case and very different channels : 1) for the end users of platforms operated by organisations (because these users will have organisation-specific questions), and 2) for the the operators of the former platforms, SaaS operators, contributors, developpers, testers,  translators, and in general all tech-savvy feedbacks and users that are concerned by the software itself (as opposed to thse who are concerned by a specific instance / custom integration).    (1) should provide in-app quick and easy ways to get information and file issues for the operating organisation only ("feedback" as support),   while (2) should focus on technical issues (of all kind, including accessibility, translation, etc.) and foster on the collaborative maintenance of a shared roadmap (besides bug-fixing of course). 
- a page or elements about the values that the software conveys, but does not explain on itself - this can be pretty technical and complex (eg. Patriot Act, US customs, etc.) so this should range for each topic from a quick all-public overview, to in-depth argumentation (details should be on Twake website, while the web or mobile app should only mention them and provide links to more advanced content)
- a page about the project history, and the current state and roadmap : because it helps users set the level of standard to evaluate the app against, and can avoid many requests and negative feedback, eg. if a wanted feature is planned and not yet released, one knows what to do (here wait or foster this development, instead of complaining, or asking for the feature to be integrating).

In a nutshell, the applications should be able to self-explain their current purpose, their roots, their goals and their path to reach it.

**Comment by Facyla** [comment]
(this issue was filed for Twake Mail, but applies in general for all Twake softwares)
